twisted/python/filepath.py

```diff
@@ -1529,8 +1529,8 @@ def setContent(self, content, ext=b'.new'):
         with sib.open('w') as f:
             f.write(content)
         if platform.isWindows() and exists(self.path):
-            os.unlink(self.path)
-        os.rename(sib.path, self.path)
+            os.remove(self.path)
```

Comment:
What's the reasoning for this change? It seems like needless churn, but it's also harmless, so I'm not going to block review on it. Generally speaking it'd be nice if we didn't make a change like this, so please consider removing it.